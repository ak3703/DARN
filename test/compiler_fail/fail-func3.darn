int foo() {}

int print() {} /* cannot use keyword print as function definition */

int main() {
	return 0;
}
