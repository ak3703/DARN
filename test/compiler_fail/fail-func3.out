Fatal error: exception Failure("Function print may not be defined")
